model-tests/jvm/src/test/scala/explore/modes/ModesSuite.scala

```diff
@@ -64,7 +64,7 @@ class ModesSuite extends CatsEffectSuite {
       )
       // .flatTap(_.traverse(IO.println))
       .map(_.length)
-      .assertEquals(85)
+      .assertEquals(48)
```

Comment:
Maybe helpful to add a couple of tests to check that the results match.  For example, to check that all results have a slit width `<= 1`?